src/atoms/Icon/Icon.native.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import icons from '../../icons';
+
+const iconSize = {
+  s: 10,
+  m: 16,
+  l: 20,
+};
+
+const Icon = ({ size, name, fill }) => {
+  const IconComponent = icons[name];
+  return <IconComponent fill={fill} height={iconSize[size]} width={iconSize[size]} />;
+};
+
+Icon.propTypes = {
+  size: PropTypes.oneOf(['s', 'm', 'l']),
+  name: PropTypes.oneOf(Object.keys(icons)),
+  fill: PropTypes.string,
```

Comment:
I'll change ```PropTypes.string``` to whatever way we decide in the Text PR

Reply:
Let's rename the fill as `color`? It remains consistent with all our namings for color as well.

Reply:
There might be more props here that we may discover later like fillOpacity, stroke, etc. These are all SVG props and we should ideally keep them the way they are going to be interpreted in the svg

Reply:
that's again flexibility over constraints 😆. Cool let's come back to this once we have the entire icon component